### PR TITLE
Import How-to related to Configuration Database

### PR DESCRIPTION
**Pull request by gwarf**
Import How-to related to Configuration Database:
* https://wiki.egi.eu/wiki/HOWTO20: Registering a new service in GOC-DB: mandatory and optional information
* https://wiki.egi.eu/wiki/HOWTO21: How to add URL service endpoint information into GOC-DB

I would be in favour of just moving this documentation under the section about the Configuration Database under https://docs.egi.eu/internal/configuration-database/, instead of keeping it as an "how to" (especially when it's just a list of required fields).

### CODE REVIEW

File: content/en/providers/operations-manuals/howto20_registering_service_gocdb_required_fields/_index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+`EGI  Local  `
+
+`✓ Reserved Tags Inheritable from Parent Site`
+`none`
+
+`✓ Reserved Tags Directly Assigned (WARNING - If deselected you will not be able to reselect the tag - it will be moved to the 'Protected Reserved Tags' list)`
+`none`
+
+`✗ Protected Reserved Tags (Can only be assigned on request)`
+`alice  atlas  cms  elixir  lhcb  tier1  tier2  wlcg`
```
Comment (enolfc) [inline content/en/providers/operations-manuals/howto20_registering_service_gocdb_required_fields/_index.md:51]:
```suggestion
`alice`  `atlas`  `cms`  `elixir`  `lhcb`  `tier1`  `tier2`  `wlcg`
```

File: content/en/providers/operations-manuals/howto20_registering_service_gocdb_required_fields/_index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+`Contact E-Mail * (valid email format)`
+
+## Scope Tags
+
+`✓ Optional Tags (At least 1 optional tags must be selected)`
+`EGI  Local  `
+
+`✓ Reserved Tags Inheritable from Parent Site`
+`none`
```
Comment (enolfc) [inline content/en/providers/operations-manuals/howto20_registering_service_gocdb_required_fields/_index.md:45]: `FedCloud` should be here no?

File: content/en/providers/operations-manuals/howto20_registering_service_gocdb_required_fields/_index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+`Is this service in production? Y/N`
+
+`Is this service monitored? Y/N`
+
+`Contact E-Mail * (valid email format)`
+
+## Scope Tags
+
+`✓ Optional Tags (At least 1 optional tags must be selected)`
+`EGI  Local  `
```
Comment (enolfc) [inline content/en/providers/operations-manuals/howto20_registering_service_gocdb_required_fields/_index.md:42]:
```suggestion
`EGI`  `Local `
```

File: content/en/providers/operations-manuals/howto20_registering_service_gocdb_required_fields/_index.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+title: HOWTO20
+weight: 135
+type: "docs"
+description: "Registering a new service in GOC-DB: mandatory and optional information"
+---
+
+Information to provide when registering a new service (the fields marked
+with an asterisk are mandatory):
+
+`Hosting Site * (select the appropriate RC under which you are registering the service)`
```
Comment (enolfc) [inline content/en/providers/operations-manuals/howto20_registering_service_gocdb_required_fields/_index.md:11]: would this look better on a table? or maybe a list?

File: content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+The information needed for service type are:
+
+  - CREAM-CE: the queue URL where the ops VO is enabled. This is the
+    value of the attribute GlueCEUniqueID published in the BDII (e.g.
+    *cream.egi.eu:8443/cream-pbs-ops*).
+  - SRM: the value of the attribute GlueServiceEndpoint published in the
+    BDII (e.g. *httpg://se.egi.eu:8444/srm/managerv2*)
+  - Other service types: the value of the attribute GlueServiceEndpoint
+    published in the BDII
+
```
Comment (enolfc) [inline content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md:25]: Missing the cloud service types info

File: content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+by setting also the "Monitored" flag, ARGO will monitor them.
+
+For having more information about managing the Service endpoints in
+GOC-DB, please consult the relative
+[documentation](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation#Service_Endpoints).
+
+## Get the information
+
+For retrieving the queue URL from the BDII, you can simply use the
+command *lcg-infosites* executed from an UI. Be sure to query a
```
Comment (enolfc) [inline content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md:36]:
```suggestion
command `lcg-infosites` executed from an UI. Be sure to query a
```

File: content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+
+`$ export LCG_GFAL_INFOSYS=egee-bdii.cnaf.infn.it:2170`
+
+`$ lcg-infosites --vo ops ce | grep nikhef`
+`  5680      15          0            0       0 dissel.nikhef.nl:2119/jobmanager-pbs-infra`
+`  5680      17          1            1       0 gazon.nikhef.nl:8443/cream-pbs-infra`
+`  5680      15          0            0       0 juk.nikhef.nl:8443/cream-pbs-infra`
+`  5680      15          0            0       0 klomp.nikhef.nl:8443/cream-pbs-infra`
+`  5680      16          0            0       0 stremsel.nikhef.nl:8443/cream-pbs-infra`
+
```
Comment (enolfc) [inline content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md:51]: better a code block

File: content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+| -------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
+| this is the home page regarding your site. You need to fill in the URL information. Click on a service for displaying its page (e.g. the CREAM-CE) | [frameless|border|800px](/File:Picture01.png "wikilink") |
+| click on the EDIT button on the top-right corner                                                                                                   | [frameless|border|800px](/File:Picture02.png "wikilink") |
+| fill in the *Service URL* field with the queue URL;                                                                                                | [frameless|border|800px](/File:Picture03.png "wikilink") |
+| Now the CREAM-CE service endpoint contains the required queue information. Proceed in a similar way for the other CREAM-CE and SRM services.       | [frameless|border|800px](/File:Picture04.png "wikilink") |
```
Comment (enolfc) [inline content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md:83]: Lots of CREAM-CE that is no longer in use

File: content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+For example:
+
+`srm://ccsrm.in2p3.fr:8443/srm/managerv2?SFN=/pnfs/in2p3.fr/data/dteam/`
+
+  - As explained in previous sections, you can retrieve the port number
+    from the *GlueServiceEndpoint* URL information.
+  - If your SE provides *GlueSAPath* information, use that. To retrieve
+    it:
+
+`$ ldapsearch -x -LLL -H `<ldap://sbdii01.ncg.ingrid.pt:2170>` -b "mds-vo-name=NCG-INGRID-PT,o=grid" '(&(objectClass=GlueSA)(GlueSAAccessControlBaseRule=VO:ops))' GlueSAPath GlueChunkKey`
```
Comment (enolfc) [inline content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md:147]: code block

File: content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md [hunk 10]
```diff
@@ -0,0 +1,175 @@
...
+
+`dn: GlueSALocalID=opsdisk:replica:online,GlueSEUniqueID=srm01.ncg.ingrid.pt,Mds-Vo-name=NCG-INGRID-PT,o=grid`
+`GlueChunkKey: GlueSEUniqueID=srm01.ncg.ingrid.pt`
+`GlueSAPath: /gstore/t2others/ops`
+
+  - if your SE doesn't provide *GlueSAPath* information, use instead the
+    *GlueVOInfoPath* one:
+
+`$ ldapsearch -x -LLL -H `<ldap://ntugrid5.phys.ntu.edu.tw:2170>` -b "Mds-Vo-name=TW-NTU-HEP,o=grid" '(&(objectClass=GlueVOInfo)(GlueVOInfoAccessControlBaseRule=VO:ops))' GlueVOInfoPath GlueVOInfoLocalID GlueChunkKey`
```
Comment (enolfc) [inline content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md:156]: code block

**Review by enolfc — CHANGES_REQUESTED** [review]
Some changes requested, since we are bringing the docs here, it's a good moment to improve them a bit.
I also think this may fit better under GOCDB documentation instead of general howto

File: content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md (hunk 7 above)
Comment (gwarf) [reply content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md]: (All this was an automatic import for whose I didn't reviewed the syntax)

File: content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md (hunk 8 above)
Comment (gwarf) [reply content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md]: Will leave CREAM until HTCondor or else info is added.

File: content/en/providers/operations-manuals/howto20_registering_service_gocdb_required_fields/_index.md (hunk 2 above)
Comment (gwarf) [reply content/en/providers/operations-manuals/howto20_registering_service_gocdb_required_fields/_index.md]: Added

File: content/en/providers/operations-manuals/howto20_registering_service_gocdb_required_fields/_index.md (hunk 4 above)
Comment (gwarf) [reply content/en/providers/operations-manuals/howto20_registering_service_gocdb_required_fields/_index.md]: Added as a list for now

File: content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md (hunk 5 above)
Comment (gwarf) [reply content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md]: Added what I found in https://docs.egi.eu/providers/cloud-compute/registration/.

**Comment by gwarf** [comment]
> Some changes requested, since we are bringing the docs here, it's a good moment to improve them a bit.
> I also think this may fit better under GOCDB documentation instead of general howto

@paolini78 OK for moving this under GOCDB documentation instead of as a general How to?

**Comment by enolfc** [comment]
If left as HOWTO, they also miss some nicer representation at the left menu (now only showing HOWTO21 and not the whole title as others)

**Comment by gwarf** [comment]
> I feel that we should keep them in the HOWTOs section, regardless if they refer to a service or another, so these howtos are all collected in a single place. In case a link can be created into the service section.

I personally feel the opposite and that having such a generic section/category is confusing and messy and going the opposite way to what the goal of the docs.egi.eu was: having a documentation site organised around service-specific section that are coherent/complete and things clearly located.
I feel that people need to find the doc where it makes most sense and ease also the management: having the Configuration Database documentation discoverable from the Configuration Database-specific section, not having things spread in the service-specific section and in other places like an How-to/Man section that would be covering only a part of the things (as all other services won't have anything there as it's very HTC specific).
In fact most of the How To could be better placed under the HTC section as they are really linked to HTC and are just noise for people caring only about cloud or DataHub.

**Comment by paolini78** [comment]
I see your point. My worry was that those manuals are mentioned into procedures, created to address an audit finding, and they could have been put in a place together other manuals. 
Let's do as you suggest ;)

File: content/en/internal/configuration-database/adding-service-endpoint/_index.md [hunk 11]
```diff
@@ -0,0 +1,236 @@
...
+weight: 30
+type: "docs"
+---
+
+For monitoring purposes, each service endpoints registered into the
+Configuration Database, and having the flags _production_ and _monitored_ should
+include the endpoint URL information in order to be contacted by the
+corresponding service-specific nagios probe.
+
+**Important: currently these steps are required only for the sites under
```
Comment (enolfc) [inline content/en/internal/configuration-database/adding-service-endpoint/_index.md:13]: Not true! specially for cloud

**Review by enolfc — COMMENTED** [review]
Looks good inside GOCDB documentation!

File: content/en/internal/configuration-database/adding-service-endpoint/_index.md (hunk 11 above)
Comment (gwarf) [reply content/en/internal/configuration-database/adding-service-endpoint/_index.md]: Sentence removed, thanks!

File: content/en/internal/configuration-database/service-information/_index.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+
+<!-- markdownlint-disable no-inline-html no-bare-urls -->
+
+| Property                                | Value                                                                              |
+| --------------------------------------- | ---------------------------------------------------------------------------------- |
+| Name                                    | Configuration Database                                                             |
+| Description                             | Central registry of the infrastructure topology                                    |
+| URL                                     | https://goc.egi.eu                                                                 |
+| Support Email                           | `gocdb-admin` `<at>` `mailman.egi.eu`                                              |
+| [Helpdesk](../../helpdesk) Support Unit | **EGI Services and Service Components** <br/> I\_\_ Configuration Database (GOCDB) |
```
Comment (enolfc) [inline content/en/internal/configuration-database/service-information/_index.md:18]: This looks broken in the rendering

**Review by enolfc — CHANGES_REQUESTED** [review]
Just a small rendering issue.

File: content/en/internal/configuration-database/service-information/_index.md (hunk 12 above)
Comment (gwarf) [reply content/en/internal/configuration-database/service-information/_index.md:18]: For me https://docs.egi.eu/documentation/329/internal/configuration-database/service-information/ looks fine, what is your issue?
Screenshot 2021-09-23 at 12 07 01

Comment (enolfc) [reply content/en/internal/configuration-database/service-information/_index.md:18]: mmm, I was thinking the `|__` was a broken table cell.

So it's good

**Review by enolfc — APPROVED** [review]

File: content/en/internal/configuration-database/service-information/_index.md (hunk 12 above)
Comment (gwarf) [reply content/en/internal/configuration-database/service-information/_index.md:18]: Maybe we could find a better UTF-8 character for this. We can always change this later.

**Comment by gwarf** [comment]
@paolini78 is this PR fine with you?

**Review by paolini78 — APPROVED** [review]
thanks for the changes

**Pull request merged**